File: pset1/mario.c
#include<stdio.h>
#include<cs50.h>

// make user promt a good input

int main(void)
{
    int n;

    do
    {
        n = get_int("size:  ");

    }

    while (n < 1);

// printing the grid

    int i;
    int j;

    for ( i = 0 ; i < n ; i++ )
    {
        for( j = 0 ; j < n ; j++ )
        {
            printf("#");
        }

        printf("\n");
    }
}

File: sentimental-mario-more/mario.py
from cs50 import get_int

# take input from rthe user for height
while True:
    h = get_int("Height: ")
    if h >= 1 and h <= 8:
        break

# print the blocks
for i in range(h):
        print(" " * (h - i - 1), end = "")
        print("#" * (i + 1), end = "  ")
        print("#" * (i + 1))
